src/buildercore/project/__init__.py

```diff
@@ -103,11 +103,20 @@ def project_list(project_locations_list=None):
     "returns a single list of projects, ignoring organization and project data"
     return project_map(project_locations_list).keys()
 
-def project_data(pname, project_locations_list=None):
-    "returns the data for a single project"
+def project_data(pname, project_locations_list=None, format_values=None):
+    "returns the data for a single project. interpolates {value} strings if `format_values` is non-nil"
```

Comment:
seems too magic to arbitrarily replace the `{value}` strings, with respect to the careful support in `buildercore.cfngen`